src/lib/statsig.ts

```diff
@@ -0,0 +1,50 @@
+import { StatSigFlags, StatsigConfigType } from '@/types/statsig';
+import { StatsigClient } from '@statsig/js-client';
+
+const fetchIp = async () => {
+  const resp = await fetch('https://api.ipify.org?format=json');
```

Comment:
can we move this into `useEndpointsConfig`? I think it's more secure to do this, also unsure if deployers want to use something diff here

Reply:
hmm we can... 🤔 but api.ipify.org is a public url so i don't think there's a security concern about exposing this

Reply:
actually i like this idea a lot - deployers can stand up their own proxy servers this way which i think is totally worth doing if they feel like it

Reply:
chatted offline; deploying without ip. sounds like we're synced about using the config for the url when we do log ip tho 🫡